modules/components/src/Aggs/AggsQuery.js

```diff
@@ -8,13 +8,9 @@ export default ({ index = '', aggs = [], ...props }) =>
   ) : (
     <Query
       name={`${capitalize(index)}AggregationsQuery`}
-      variables={{
```

Comment:
is this on purpose?

Reply:
This is part of the normalization commit